src/libraries/xchaininit/src/xchain_command.cc

```diff
@@ -437,7 +437,7 @@ int parse_execute_command(const char * config_file_extra, int argc, char * argv[
     std::string miner_type;
     std::string miner_name;
     registerMiner_app->add_option("top_num", registerMiner_amount, "Miner register deposit,unit is TOP.")->required();
-    registerMiner_app->add_option("miner_type", miner_type, "Miner type: edge, validator, advance, full_node.")->required();
+    registerMiner_app->add_option("miner_type", miner_type, "Miner type: edge, validator, advance, archive, exchange.")->required();
```

Comment:
no fullnode?

Reply:
> no fullnode?

fullnode is a node type, it is from Advance Miner Type. We don't have fullnode Miner Type.